src/components/select/SelectCommons.tsx

```diff
@@ -102,60 +109,71 @@ export const SelectBase = forwardRef<HTMLInputElement, PropsWithLabels<SelectBas
     });
 
     return (
-      <SelectStyled
-        allowDeselect={isClearable}
-        aria-label={ariaLabel}
-        aria-labelledby={ariaLabelledBy}
-        classNames={
-          hasDropdownAutoWidth
-            ? { ...SELECT_CLASSNAMES, dropdown: 'echoes-select-dropdown-auto-width' }
-            : SELECT_CLASSNAMES
-        }
-        clearButtonProps={
-          isClearable
-            ? {
-                'aria-label': intl.formatMessage({
-                  id: 'select.clear',
-                  defaultMessage: 'Clear select field',
-                  description:
-                    'Screen reader-only text to indicate that the select field can be cleared with a button',
-                }),
-                className: 'echoes-select-close-button',
-                icon: <IconX />,
-              }
-            : {}
-        }
-        clearable={isClearable}
-        comboboxProps={{
-          portalProps: {
-            target: portalContext.portalReference,
-          },
-          withinPortal: isDefined(portalContext.portalReference),
-        }}
-        data={data}
-        data-variant={highlight}
+      <FormField
+        controlId={controlId}
         description={helpText}
```

Comment:
The description is not bound with aria-describedBy anymore, same for the error and valid message.

Reply:
I added `aria-describedby` to the select component but mantine select doesn't seem to be forwarding it to the input component.

Reply:
Arf maybe that means we need to keep using the `description` prop of mantine for that and just re-use the style of FormFieldMessage?

Reply:
Then we would have to do that with the validation messages as well... I wonder if I can just get a ref to the underlaying input and add the aria attributes imperatively as a workaround?

Reply:
True, that sounds like a good workaround if you can make it work

Reply:
Seems to be working https://github.com/SonarSource/echoes-react/pull/247/commits/e357063e5bd80d4ba083f212006a7cc443adb967

Reply:
Nice!

Reply:
It might be worth extracting the `setRef` helper to a common folder, I believe we might need to get refs internally in more component in the future, it would also probably be nice to make it a bit more explicit in the name that it's keeping a ref for ourself and still setting the one we receive from the parent.

Reply:
See https://github.com/SonarSource/echoes-react/pull/247/commits/e155bb912d25cff943b20fa68198814ec7728e93